packages/marko-web-native-x/components/story/track-end-of-content.marko

```diff
@@ -0,0 +1,19 @@
+import defaultValue from "@parameter1/base-cms-marko-core/utils/default-value";
+import { get, getAsObject } from "@parameter1/base-cms-object-path";
+
+$ const { req } = out.global;
+$ const story = getAsObject(input, "story");
+$ const push = defaultValue(input.push, false);
+
+<if(push)>
+  <marko-web-gtm-push name="dataLayerNativeX" data={
```

Comment:
This should probably be it's own component should be explicitly called by the user before calling any of these tracking components